Automattic/woocommerce-payments
# Added woo updater code and hash and updated release task

Fixes #385

#### Changes proposed in this Pull Request

* Adds woo-updater code
* Adds woo hash
* Updates release task to include woo-includes

#### Testing instructions

* npm run release
* Ensure the resulting ZIP installs and activates on a test site without issues
